esmvalcore/preprocessor/_area.py

```diff
@@ -203,16 +202,12 @@ def average_region(cube, coord1, coord2, operator='mean', fx_files=None):
 
     Parameters
     ----------
-    cube: iris.cube.Cube
-        input cube.
-    coord1: str
-        Name of the firct coordinate dimension
-    coord2: str
-        Name of the second coordinate dimension
-    operator: str
-        Name of the operation to apply (default: mean)
-    fx_files: dict
-        dictionary of field:filename for the fx_files
+        cube: iris.cube.Cube
+            Input cube.
+        operator: str
+            Name of the operation to apply
```

Comment:
Mention what values can be used for this argument

Reply:
```suggestion
            The operation, options: mean, median, min, max, std_dev, variance
```
Also note that there is a full table describing each operation in this functions documentation.